src/assets/toolkit/styles/components/message.css

```diff
@@ -0,0 +1,38 @@
+/** @define Message */
+
+:root {
+  --Message-padding: var(--space-xs) 2.5em;
+  --Message-background: var(--color-gray);
+  --Message-offline-background: var(--color-orange);
+  --Message-offline-color: var(--color-white);
```

Comment:
If it's just barely failing, I'm not concerned. Human beings don't conform to a concrete numeric scale... I think it's fine to give ourselves a little wiggle room.

(Though if you ever have suggested color adjustments to the palette that will make them more attractive _and_ readable, I encourage those proposed edits site-wide!)